superbench/runner/runner.py

```diff
@@ -3,9 +3,13 @@
 
 """SuperBench Runner."""
```

Comment:
After reading the code, this integration should be separated from this PR. We should write UT to make sure class `AnsibleClient` and related playbook work. After this PR merged, we can create PR for this integration.

Reply:
After discussion, we can add PR for UT separately.